determined-ai/determined
# feat: add `last seen` column in user management table

## Description
[WEB-1712]

Frontend change is to add `Last Seen` columns and support sort for all columns

Backend change is to make `last_log` optional and add sort for `last_login`. `last_login` should be renamed as `last_auth_at` or something similar but not in this PR




## Test Plan
- Check if `Last Seen` (last authenticated time) column is added in user management table
  - When `Last Seen` is not available, `N/A` is on display
- Check if each column is sortable by clicking header




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-1712]: https://hpe-aiatscale.atlassian.net/browse/WEB-1712?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
